### DISCUSSION TITLE
230907 홍보 전략글 확정 회의 (16:00 ~ 16:10)

### DISCUSSION THREAD

**Discussion by somsom13** [woowacourse-teams/2023-shook#364] [회의]
## 회의 안건
9/8까지 제출해야 하는 홍보 전략 계획 글을 확정한다.
- 의견이 있다면 해당 디스커션에 댓글로 작성한다.

# S-HOOK 홍보 계획 글

## 사용자 유치 계획

노래 추천이 가능하고, 좋아하는 노래 정보를 공유할 수 있는 커뮤니티에 홍보

### 1. 에브리타임, 네이버 지식인, 오픈 채팅방 등 커뮤니티 홍보

노래 추천 글이나 게시판, 채팅방 같은 커뮤니티에 서비스로 접근할 수 있는 링크를 공유한다. 멜론이나 특정 음원 사이트가 아닌 S-HOOK 서비스에서 노래를 들을 수 있는 url 을 공유하여 홍보할 예정이다.

### 2. 노래 추천 오픈 채팅방 운영

현재 활성화 되어 있는 오픈 채팅방에서는 보통 유튜브 링크나 카카오 뮤직, 멜론 링크를 사용하는 경우가 많다. 그래서 S-HOOK 의 링크를 공유하여 홍보하기 어려운 오픈 채팅방들도 존재한다.

일시적이지 않고 지속적으로 S-HOOK 서비스를 중심으로 노래를 공유할 수 있는 창구를 만들기 위해 오픈 채팅방을 운영하여 서비스를 홍보할 예정이다.

### (3. 서비스 홍보를 위한 트위터 계정 운영)

인스타그램은 광고를 하지 않으면 일반 사용자의 알고리즘에 노출되기 어렵다. 또한 인스타그램 특성상, S-HOOK을 소개하는 계정이 활성화되기 어려운 부분이 있다.

트위터는 K-POP 팬들뿐만 아니라 음악을 좋아하는 팬들이 많이 존재한다. 계정의 팔로워가 적더라도 게시글 공유가 활발한 트위터의 특성 상, 한 번만 노출이 되어도 더 많은 사용자들에게 노출될 가능성이 높다.

멜론 탑 100 노래들을 제공할 예정이기 때문에, 멜론 차트 순위에 존재하는 노래 킬링파트를 빨리 들을 수 있다는 특징을 강조해서 홍보할 예정이다.

## 지속적인 홍보를 위한 계획

- 팀에서 운영하는 오픈 채팅방, SNS 계정을 사용하여 지속적으로 서비스를 사용자들에게 알린다.
- 멜론 탑 100 노래를 주기적으로 업데이트하여 최신 인기 노래들의 킬링파트들을 통해 사용자가 노래를 빠르게 접할 수 있도록 한다.

## 예상 시간
1시간

**Comment by Cyma-s** [comment]
# 홍보 계획 글

## 사용자 유치 계획

### 1. 에브리타임, 오픈 채팅방 등 커뮤니티 홍보

노래 추천 글이나 게시판, 채팅방 같은 커뮤니티에 서비스로 접근할 수 있는 링크를 공유한다. 멜론이나 특정 음원 사이트가 아닌 S-HOOK 서비스에서 노래를 들을 수 있는 url 을 공유하여 홍보할 예정이다.

### 2. 노래 추천 오픈 채팅방 운영

현재 활성화 되어 있는 오픈 채팅방에서는 보통 유튜브 링크나 카카오 뮤직, 멜론 링크를 사용하는 경우가 많다. 그래서 S-HOOK 의 링크를 공유하여 홍보하기 어려운 오픈 채팅방들도 존재한다.

일시적이지 않고 지속적으로 S-HOOK 서비스를 중심으로 노래를 공유할 수 있는 창구를 만들기 위해 오픈 채팅방을 운영하여 서비스를 홍보할 예정이다.

### 3. 서비스 홍보를 위한 트위터 계정 운영

인스타그램은 광고를 하지 않으면 일반 사용자의 알고리즘에 노출되기 어렵다. 또한 인스타그램 특성상, S-HOOK을 소개하는 계정이 활성화되기 어려운 부분이 있다.

트위터는 K-POP 팬들뿐만 아니라 음악을 좋아하는 팬들이 많이 존재한다. 계정의 팔로워가 적더라도 게시글 공유가 활발한 트위터의 특성 상, 한 번만 노출이 되어도 더 많은 사용자들에게 노출될 가능성이 높다.

멜론 탑 100 노래들을 제공할 예정이기 때문에, 멜론 차트 순위에 존재하는 노래 킬링파트를 빨리 들을 수 있다는 특징을 강조해서 홍보할 예정이다.

### 4. 서비스 홍보를 위한 유튜브 채널 운영

서비스를 통해 만들어진 킬링파트를 알려주고 이를 소비하는 것을 영상에 담아 서비스를 노출시킨다.

- 장점
    - 유튜브는 킬링파트라는 단어에 대한 노출, 킬링파트 형식의 미디어에 대한 노출이 가장 많은 곳이다.
        - 유저들이 킬링파트라는 것에 대한 이해가 되어있을 확률이 높다.
        - 유저들이 킬링파트를 소비하는 것이 익숙할 확률이 높다.
    - 외부 커뮤니티에 의존하지 않기에 지속적인 홍보 방식을 정할 수 있다.
        - 커뮤니티의 경우 활성화 정도가 언제든지 변할 수 있다.
        - 커뮤니티의 경우 유저들의 성향, 맞춤 홍보방식에 대한 고려가 필요하다.
- 홍보 방식
    1. 영상에 슉의 킬링파트의 위치( ex 10초 ~ 15초 ) 만 알려준다.
    2. 서비스에서 정해진 킬링파트 3개를 이어 붙인 *영상 을 만들어서 들을 수 있게 하면서 서비스를 알린다.

*영상 : 동영상 or 쇼츠

## 홍보를 지속하기 위한 계획

- 팀에서 운영하는 오픈 채팅방, SNS 계정을 사용하여 지속적으로 서비스를 사용자들에게 알린다.
- 최신 인기 노래를 주기적으로 업데이트하여 사용자가 최신 노래를 빠르게 접할 수 있도록 한다.
